src/+otp/+qg/QuasiGeostrophicParameters.m

```diff
@@ -10,5 +10,9 @@
         Re %MATLAB ONLY: (1,1) {mustBeFinite, mustBePositive} = 450
         %Ro is the Rossby number
         Ro %MATLAB ONLY: (1,1) {mustBeFinite} = 0.0036
+        %ADlambda is the approximate deconvolution parameter
+        ADLambda %MATLAB ONLY: (1,1) {mustBeFinite, mustBePositive} = 1
```

Comment:
I'm not familiar with this approximate filter, but is there a set of params that allow it to be disabled?  Unless it is very commonly used, I don't want to force approximations to the underlying PDE

Reply:
Nevermind, it's in a separate RHS.  Didn't review far enough.  This might cause some confusion when params affect one RHS but not another.

Reply:
>This might cause some confusion when params affect one RHS but not another.

I'm not sure of a good way to solve this issue. This will not be the last problem that has weird RHS issues I assume, so finding a good compromise right now is key IMO.